developments_build/sql/_status.sql

```diff
@@ -52,113 +52,82 @@ IN PREVIOUS VERSION:
-                THEN '5. Complete'
-
-            WHEN a.x_withdrawal IN ('W', 'C')
-                THEN '9. Withdrawn'
-
-            WHEN date_statusp IS NOT NULL
-                THEN '2. Plan Examination'
-
-            WHEN date_permittd IS NOT NULL
-                THEN '3. Permitted'
-
-            ELSE status_translate(a._job_status)
-        END) as job_status
+        a._complete_year,
+        a._complete_qrtr
     FROM _MID_devdb a
-),
-DRAFT_STATUS_devdb as (
-    SELECT
-        job_number,
-        job_type,
-        job_status,
-        date_permittd,
-        date_lastupdt::date,
-        classa_net,
-        address,
-        occ_proposed,
-        date_complete,
-        -- update year_compelete based on job_type and status
-        (CASE
-            WHEN job_type = 'Demolition'
-                OR job_status NOT IN ('4. Partial Complete', '5. Complete')
-                THEN NULL
-            ELSE _complete_year
-        END) as complete_year,
-
-        -- update complete_qrtr based on job_type and job_status
-        (CASE
-            WHEN job_type = 'Demolition'
-                OR job_status NOT IN ('4. Partial Complete', '5. Complete')
-                THEN NULL
-            ELSE _complete_qrtr
-        END) as complete_qrtr,
-
-        -- Assign classa_complt based on job_status
-        (CASE
-            WHEN job_status = '5. Complete' 
-                THEN classa_net
-            WHEN job_status = '4. Partial Complete' 
-                THEN co_latest_units
-            ELSE NULL
-        END) as classa_complt,
-
-        -- Assing classa_incmpl
-        (CASE
-            WHEN job_status = '5. Complete' 
-                THEN NULL
-            WHEN job_status = '4. Partial Complete'
-                THEN classa_net-co_latest_units
-            ELSE classa_net
-        END) classa_incmpl
-    FROM STATUS_translate
+    JOIN dob_jobapplications b
+    ON a.job_number::int = b.jobnumber::int
 )
 SELECT
     job_number,
     job_type,
     job_status,
     date_lastupdt,
     date_permittd,
-    complete_year,
-    complete_qrtr,
-    classa_complt,
-    classa_incmpl,
+    _complete_year as complete_year,
+    _complete_qrtr as complete_qrtr,
+
+    -- Assign classa_complt based on job_status
+    (CASE
+        WHEN job_status = '5. Completed Construction' 
+            THEN classa_net
+        WHEN job_status = '4. Partially Completed Construction' 
+            THEN co_latest_units
+        ELSE NULL
+    END) as classa_complt,
+
+    -- Assing classa_incmpl
+    (CASE
+        WHEN job_status = '5. Completed Construction' 
+            THEN NULL
+        WHEN job_status = '4. Partially Completed Construction'
+            THEN classa_net-co_latest_units
+        ELSE classa_net
+    END) as classa_incmpl,
+
     classa_net,
     address,
     occ_proposed,
     (CASE 
         WHEN date_complete IS NOT NULL 
             THEN NULL
         WHEN (CURRENT_DATE - date_lastupdt)/365 >= 2 
```

Comment:
should we consider switching out `CURRENT_DATE` to a variable? 
since reruns might produce different results?

Reply:
Good idea. I opened a new issue for this.

Reply:
I agree that CURRENT_DATE should be a variable.  Future enhancement.